RabbitMQ/docker-compose.yml

```diff
@@ -11,7 +11,6 @@ services:
       - "5672:5672"
   rabbitmq-config:
     container_name: rabbitmq-config
-    image: ${PRIVATE_REGISTRY_URI}mushkevych/syn-rabbitmq-config:latest
```

Comment:
Previously when test, the docker image that is pulled from that would cause issues with the config not being able to start. By creating the image using the dockerfile, the issue was removed. Anyways i dont think this is an issue right? cause we are using the deployed rabbitmq

Reply:
that's weird, were you running this instance on your local?

this repo is used by the remote instance to spin up the exact same environment so changing it will actually also affect the remote rabbitmq instance

Reply:
before converting to the remote RabbitMQ, i was running it locally. but now im just using the remote instance

Reply:
in that case, if there's no need to change anything on the RabbitMQ folder, and since remote is dependent on it, could I trouble you to revert those changes? thanks!

Reply:
done